certified-connectors/RecordedFutureSandbox/apiDefinition.swagger.json

```diff
@@ -177,7 +177,7 @@
                   "type": "array"
                 },
                 "tasks": {
-                  "type": "string"
+                  "type": "object"
```

Comment:
@lindarecfut Typically this will be considered as a breaking change and it is not allowed. but since this is a new connector and still there are no flows created using it then we will ignore the below error messages.
```
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/GetReport/responses/200/schema/tasks/type Source: NewSwagger
```